src/util/splitTextByTerm/splitTextByTerm.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { splitTextByTerm } from './splitTextByTerm'
+
+describe('splitTextByTerm', () => {
```

Comment:
What if:

```typescript
splitTextByTerm('hello world', '\x') // 💣💥
```

Reply:
Thanks for pointing this out! This is not an error that can be caught or handled at runtime, because it prevents your code from running at all.
If the search term comes from some source other than a user input field, it's the responsibility of the code that generates the search term to ensure that it's a valid string.

Invalid escape sequences are now handled from the function `replaceInvalidEscapeSequences` in [4a6638e](https://github.com/okp4/dataverse-portal/pull/266/commits/4a6638e01011090eb5e56f07b1a59cbfaa0c4a58).